phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java

```diff
@@ -148,11 +148,11 @@ public Void presentationHtml(String htmlToShow) {
     @Override
     public Void presentation(String programName, String programEditor, String applicationHomepage,
             String scriptorName) {
-        final String htmlToShow = "<body>" + tr("This wizard will help you install {0} on your computer", programName)
-                + ".<br><br>" + tr("This program was created by: {0}", programEditor) + "<br><br>"
+        final String htmlToShow = "<body>" + tr("Installation Wizard for {0}", programName)
```

Comment:
Why uppercase wizard and script?

Reply:
English strings in computer usage seem to be written like this. Can make them lowercase.
https://en.wikipedia.org/wiki/Camel_case

Reply:
I would prefer lower case here. It's not a name or anything like that.

Reply:
done